tests/pcmsolver/scf/input.dat

```diff
@@ -63,6 +63,14 @@ energy_scf, wfn = energy('scf', return_wfn=True)
 compare_values(totalenergy, energy_scf, 10, "Total energy (PCM, separate algorithm)") #TEST
 compare_values(polenergy, wfn.get_variable("PCM POLARIZATION ENERGY"), 6, "Polarization energy (PCM, separate algorithm)")  #TEST
 
+# Now force use of ROHF on NH3 to check sanity of the algorithm with PCM
+set pcm_scf_type total
+set reference rohf
+print('PK-ROHF-PCM, total algorithm')
+energy_scf, wfn = energy('scf', return_wfn=True)
+compare_values(totalenergy, energy_scf, 10, "Total energy (PCM, separate algorithm)") #TEST
+compare_values(polenergy, wfn.get_variable("PCM POLARIZATION ENERGY"), 6, "Polarization energy (PCM, separate algorithm)")  #TEST
```

Comment:
`get_variable` --> `variable` to fend off the warning. here and below.

Reply:
Sure thing :smile_cat: